src/web/app/pages-instructor/student-profile/student-profile.component.ts

```diff
@@ -12,26 +13,29 @@ import { StudentProfile } from './student-profile';
   styleUrls: ['./student-profile.component.scss'],
 })
 export class StudentProfileComponent implements OnInit {
-
   Gender: typeof Gender = Gender; // enum
+  photoUrl?: string;
```

Comment:
In this case, photoUrl can be an `@Input()` property?

An component should not have any assumption about the pages use them.

Reply:
Yup! I have made it have a default value instead of being an optional. This isnt exactly an `@Input()` since it is always there.